lib/src/directives/language_direction/language_direction_directive.dart

```diff
@@ -0,0 +1,28 @@
+import 'dart:html';
+import 'package:angular2/core.dart';
+import 'package:intl/intl.dart' as intl;
+
+@Directive(
+  selector: "[skawaLangugageDirection]",
+  exportAs: 'skawaLangugageDirection',
+)
+class LanguageDirectionDirective {
```

Comment:
The angular_components has nothing like this. Its has ang rtl annotations based an the `document`.